products/models.py

```diff
@@ -0,0 +1,47 @@
+"""
+Models for the products app
+"""
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from users.models import User
+
+
+class TimeStamp(models.Model):
+    """
+    TimeStamp model for product class
+    """
+
+    created_on = models.DateTimeField(auto_now_add=True)
+    updated_on = models.DateTimeField(auto_now=True)
+    created_by = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    class Meta:
+        """
+        Defines the metadata of the class
+        """
+
+        abstract = True
+
+
+class Product(TimeStamp):
+    """
+    Product model
+    """
+
+    price = models.IntegerField(_("Product Price"))
+    quantity = models.IntegerField(_("Product Quantity"))
```

Comment:
The better representations should be `stock_quantity`

Reply:
Done